pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -312,6 +309,8 @@ func handleNetworkInstanceCreate(
 			VlanMap:       make(map[uint32]uint32),
 		},
 	}
+	appNumOnUNetBaseCreate(status.UUID)
+	status.DhcpRange.Size = getAppCountOnUNet(ctx, &status)
```

Comment:
I think the setting of DhcpRange.Size needs to be in parseIpspec.
Alternatively have a Size() member function which calculates the size when called.

Reply:
In fact you already have such a function called HostCount(). So you can delete the Size field.